Getting-Started.md

```diff
@@ -398,6 +399,48 @@ starting/(re)creating/building/attaching containers for a service.
 	```
     And restart the system.
 
+
+#### SLL error [&uarr;](#ssl-)
+
+* Indication: localhost:9200 refuses connection error.
+
+* Diagnosis: 
+
+![ssl_error](https://user-images.githubusercontent.com/53489999/77805355-aea32c00-70a7-11ea-9222-5073851bdc60.png)
+
+* Solution:
+
+  * Change 'https' to 'http' in the setup.cfg file
+
+  ```
+  [es_collection]
+  # arthur = true
+  # arthur_url = http://127.0.0.1:8080
+  # redis_url = redis://localhost/8
+  url = http://localhost:9200
+
+  [es_enrichment]
+  url = http://localhost:9200
+  ```
+
+
+#### cloc installation [&uarr;](#cloc-installation-)
+
+* Diagnosis:
+
+![cloc_error](https://user-images.githubusercontent.com/53489999/77805464-f45ff480-70a7-11ea-90ef-04e1c588f2fd.png)
```

Comment:
Can you replace the figure with the text: `[Errno 2] No such file or directory ...`? thanks